packages/backend/src/plugins/catalog.ts

```diff
@@ -33,6 +33,7 @@ export default async function createPlugin(
     eventBroker: env.eventBroker,
   });
   builder.addEntityProvider(demoProvider);
+  builder.setEventBroker(env.eventBroker);
```

Comment:
No need to add this right? since it's not hooked up to anything. That way this change will be isolated to the catalog

Reply:
You are right. Done thanks.